distribution/docs/src/main/resources/_contents/documentationnotescontents.adoc

```diff
@@ -0,0 +1,72 @@
+The {branding} documentation is segmented by user needs, with users categorized as Users, Administrators, Integrators, and Developers.
+
+=== Documentation Outline
+
+The documentation is segmented by user needs, with users categorized as Users, Administrators, Integrators, and Developers. 
+
+Users::
+Users are end users interacting with the applications at the most basic level.
+User help is available through the *Help* link in the User Interface.
+
+Administrators::
+<<Managing {branding}>> |
+Administrators will be installing, maintaining, and supporting existing applications.
+Sections titled with Managing support administrators.
+
+Integrators::
+<<Integrating {branding}>> |
+Integrators will use the existing applications to support their external frameworks.
+Sections titled with Integrating support administrators.
+
+Developers::
+<<Extending {branding}>> |
+Developers will build or extend the functionality of the applications. 
+Sections titled with Extending support administrators.
+
+==== Documentation Updates
+The most current Distributed Data Framework {branding}) documentation is available at: https://codice.atlassian.net/wiki/display/DDF/DDF+Documentation[{branding} Documentation].
+
+==== Conventions
+
+The following conventions are used within this documentation:
+
+===== Callouts
+
+[TIP]
+====
+This is a *Tip*, used to provide helpful information.
+====
+
+[NOTE]
+====
+This is an *Informational Note*, used to emphasize points, remind users of beneficial information, or indicate minor problems in the outcome of an operation.
+====
+
+[WARNING]
+====
+This is an *Emphasized Note*, used to inform of important information.
+====
+
```

Comment:
kind of? Asciidoctor switches the icons from how Confluence uses them, but I think we are ready to switch to a pure Asciidoctor style.